### ISSUE TITLE
Can I remove "Performing Streamed Install" command inside sndcpy.bat without getting an error?

### ISSUE THREAD

**Issue by Simplegram** [rom1v/sndcpy#145]
Can I remove install apk command without the sndcpy terminal quitting? Without performing streamed install sndcpy gets significantly faster to launch side by side with scrcpy.

**Comment by PipGuy** [comment]
Yes you can remove it safely. However you need to activate the app manually. In my case i activate the app first then i plug it in and launch sndcpy and scrcpy
